dataprep/eda/missing/compute/nullivariate.py

```diff
@@ -14,42 +14,115 @@
 from ...intermediate import Intermediate
 from ...staged import staged
 
+__most__ = 5
+__longest__ = 5
+
 
 def _compute_missing_nullivariate(df: DataArray, bins: int) -> Generator[Any, Any, Intermediate]:
     """Calculate the data for visualizing the plot_missing(df).
     This contains the missing spectrum, missing bar chart and missing heatmap."""
+    # pylint: disable=too-many-locals
 
     df.compute()
 
     nullity = df.nulls
     null_cnts = nullity.sum(axis=0)
     nrows = df.shape[0]
+    ncols = df.shape[1]
     null_perc = null_cnts / nrows
+    miss_perc = nullity.sum() / (nrows * ncols)
+    avg_row = nullity.sum() / nrows
+    avg_col = nullity.sum() / ncols
 
     tasks = (
         missing_spectrum(df, bins=bins),
         null_perc,
         missing_bars(null_cnts, df.columns.values, nrows),
         missing_heatmap(df),
         missing_dendrogram(df),
+        nullity.sum(),
+        missing_col_cnt(df),
+        missing_row_cnt(df),
+        missing_most_col(df),
+        missing_most_row(df),
+        miss_perc,
+        avg_row,
+        avg_col,
     )
 
     ### Lazy Region End
-    spectrum, null_perc, bars, heatmap, dendrogram = yield tasks
+    (
+        spectrum,
+        null_perc,
+        bars,
+        heatmap,
+        dendrogram,
+        cnt,
+        col_cnt,
+        row_cnt,
+        most_col,
+        most_row,
+        miss_perc,
+        avg_row,
+        avg_col,
+    ) = yield tasks
     ### Eager Region Begin
 
     sel = ~((null_perc == 0) | (null_perc == 1))
     heatmap = pd.DataFrame(
         data=heatmap[:, sel][sel, :], columns=df.columns[sel], index=df.columns[sel]
     )
 
+    if most_col[0] <= __most__:
+        top_miss_col = (
+            str(most_col[0])
+            + "-col(s) "
+            + str("(" + ", ".join(abbr(df.columns[e]) for e in most_col[2]) + ")")
+        )
+    else:
```

Comment:
This part is a little redundant. There are some suggestions.
1. `abbr(df.columns[e]) for e in most_col[2]` may be write as `df.columns[most_col[2]].map(abbr)`, which looks simpler. 
2. you could use a variable to save the suffix. E.g.,  `suffix = "" if most_col[0]<=__most__ else ", ..."`. Then the last str are `str("(" + ", ".join(abbr(df.columns[e]) for e in most_col[2]) + suffix + ")")` for both conditions and `top_miss_col` only need to be write once. 

The same comments also hold for processing of `top_miss_row`